ansible/roles/test/vars/testcases.yml

```diff
@@ -213,7 +213,7 @@ testcases:
 
     crm:
       filename: crm.yml
-      topologies: [t1, t1-lag, t0, t0-56, t0-64, t0-116]
+      topologies: [t0-8, t1, t1-lag, t0, t0-56, t0-64, t0-116]
```

Comment:
We’ll want to support this running on t1-8 also, since that’s the other one we have available.